manifests/10_deployment-ibm-cloud-managed.yaml

```diff
@@ -1,4 +1,3 @@
-# *** AUTOMATICALLY GENERATED FILE - DO NOT EDIT ***
```

Comment:
Why did you remove this comment?

Reply:
I did not, I run `make update` after I changed the `yamls` 

btw, does the `yaml` need a new line at the end ?

Reply:
`make verify` does not like this change.

Reply:
Yes, this line was removed after executing `make update`. 
So I have `manually` restored the file, after executing `make update`, `make`, and `make test`